import{r as e,H as t,a,b as n,S as r,c as l,d as o,e as i,f as s,g as c,E as m,h as p,i as h,j as d,G as u,R as g,C as E,A as b,k as f,l as w,m as y,n as k,o as v}from"./vendor.a7428e9b.js";function x(){const[u,g]=e.useReducer((e=>!e),!1),E=e.createElement(e.Fragment,null,e.createElement(t,{"aria-label":"Elsewhere",menuLinkName:"elsewhere"},e.createElement(a,{href:"https://www.linkedin.com/in/keller-mark/"},"LinkedIn"),e.createElement(a,{href:"https://scholar.google.com/citations?user=hnKZ5mwAAAAJ"},"Google Scholar"),e.createElement(a,{href:"https://github.com/keller-mark"},"GitHub"),e.createElement(a,{href:"https://beta.observablehq.com/@keller-mark"},"Observable"),e.createElement(a,{href:"https://vis.social/@mkeller7",rel:"me"},"Mastodon"),e.createElement(a,{href:"https://soundcloud.com/keller-mark-7"},"SoundCloud")));return e.createElement(n,{className:"bg-gradient2"},e.createElement(r,null),e.createElement(l,{"aria-label":"Open menu",onClick:()=>g(),isActive:u}),e.createElement(o,{href:"#/",prefix:""},"Mark Keller"),e.createElement(i,{className:"mk--lg-header-nav"},E),e.createElement(s,null,e.createElement(c,{"aria-label":"mark_keller[at]hms.harvard.edu","data-email":"true",onClick:()=>{document.querySelector('.bx--header__action[data-email="true"]').focus()},tooltipAlignment:"end"},e.createElement(m,null))),e.createElement(p,{"aria-label":"Side navigation",expanded:u,isPersistent:!1},e.createElement(h,null,e.createElement(d,null,E))))}!function(e=".",t="__import__"){try{self[t]=new Function("u","return import(u)")}catch(a){const n=new URL(e,location),r=e=>{URL.revokeObjectURL(e.src),e.remove()};self[t]=e=>new Promise(((a,l)=>{const o=new URL(e,n);if(self[t].moduleMap[o])return a(self[t].moduleMap[o]);const i=new Blob([`import * as m from '${o}';`,`${t}.moduleMap['${o}']=m;`],{type:"text/javascript"}),s=Object.assign(document.createElement("script"),{type:"module",src:URL.createObjectURL(i),onerror(){l(new Error(`Failed to import: ${e}`)),r(s)},onload(){a(self[t].moduleMap[o]),r(s)}});document.head.appendChild(s)})),self[t].moduleMap={}}}("/assets/");const C=[{date:"July 13, 2022",slug:"ismb-polyphony-biovis",description:e.createElement("p",null,"Presented Polyphony at the ISMB ",e.createElement("a",{href:"https://www.iscb.org/ismb2022-program/abstracts/biovis"},"BioVis COSI"),", winning the best long abstract award.")},{date:"July 11, 2022",slug:"ismb-vitessce-tech-track",description:e.createElement("p",null,"Presented Vitessce at the ISMB ",e.createElement("a",{href:"https://www.iscb.org/ismb2022-program/tech-track"},"Technology Track"),".")},{date:"June 16, 2022",slug:"cistrome-explorer-preprint",description:e.createElement("p",null,"Cistrome Explorer preprint uploaded to ",e.createElement("a",{href:"https://osf.io/asbz2/"},"OSF Preprints"),".")},{date:"April 6, 2022",slug:"keystone-scb-2022",description:e.createElement("p",null,"Presented at Keystone Symposium on ",e.createElement("a",{href:"https://www.keystonesymposia.org/conferences/conference-listing/meeting?eventid=6802"},"Single Cell Biology")," in Florence.")},{date:"March 31, 2022",slug:"polyphony-preprint",description:e.createElement("p",null,"Polyphony preprint uploaded to ",e.createElement("a",{href:"https://osf.io/b76nt/"},"OSF Preprints"),".")},{date:"January 19, 2022",slug:"big-pqe",description:e.createElement("p",null,"Completed preliminary qualifying exam (",e.createElement("a",{href:"https://bigphd.hms.harvard.edu/program/pqe"},"PQE"),").")},{date:"October 18, 2021",slug:"vitessce-preprint",description:e.createElement("p",null,"Vitessce preprint uploaded to ",e.createElement("a",{href:"https://osf.io/y8thv/"},"OSF Preprints"),".")},{date:"September 2, 2020",slug:"start-gradschool",description:e.createElement("p",null,"First day of PhD program.")}],M=[{title:"Vitessce",slug:"vitessce",img:"vitessce.png",description:e.createElement("p",null,"Since 2019, I have worked with the Gehlenborg Lab on ",e.createElement("a",{href:"http://vitessce.io/"},"Vitessce"),", an interactive visualization tool for spatial single-cell multi-omics experiments. Vitessce can be used as a ",e.createElement("a",{href:"https://github.com/vitessce/vitessce"},"React component"),", ",e.createElement("a",{href:"https://github.com/vitessce/vitessce-python"},"Jupyter widget"),", or ",e.createElement("a",{href:"https://github.com/vitessce/vitessceR"},"R htmlwidget"),", and the React component has been integrated into the ",e.createElement("a",{href:"https://portal.hubmapconsortium.org/"},"HuBMAP data portal"),". This work was presented as a poster at BioVis@ISMB 2019 and BioVis@ISMB 2020.")},{title:"Polyphony",slug:"polyphony",img:"polyphony.png",description:e.createElement("p",null,"In 2022, I contributed to the development of ",e.createElement("a",{href:"https://github.com/scPolyphony"},"Polyphony"),", an interactive transfer learning framework for single-cell data analysis.")},{title:"Cistrome Explorer",slug:"cistrome-explorer",img:"cistrome-explorer.png",description:e.createElement("p",null,"Prior to starting graduate studies, I worked with the ",e.createElement("a",{href:"http://gehlenborglab.org"},"Gehlenborg Lab")," on ",e.createElement("a",{href:"http://cisvis.gehlenborglab.org"},"Cistrome Explorer"),", a wrapper component for ",e.createElement("a",{href:"http://higlass.io"},"HiGlass")," with features for viewing ",e.createElement("a",{href:"http://cistrome.org/db/"},"CistromeDB")," data and querying ",e.createElement("a",{href:"http://toolkit.cistrome.org/"},"CistromeDB Toolkit"),". This work was presented as a poster at BioVis@ISMB 2020, NCI ITCR 2020, and Genome Informatics 2020.")},{title:"4CE",slug:"forsee",img:"4ce.png",description:e.createElement("p",null,"During the COVID-19 pandemic, I contributed to ",e.createElement("a",{href:"https://github.com/covidclinical/visualization-notebooks"},"data visualization & analysis")," and ",e.createElement("a",{href:"https://github.com/covidclinical/website"},"web development")," with the Consortium for Clinical Characterization of COVID-19 by EHR (",e.createElement("a",{href:"http://covidclinical.net"},"4CE"),").")},{title:"BEAT-PD",slug:"beat-pd",img:"beat-pd.png",description:e.createElement("p",null,"During the summer of 2020, I participated in the ",e.createElement("a",{href:"https://www.braincommons.org/beat-pd-dream-challenge-winners-announced/"},"BEAT-PD DREAM Challenge")," on winning team dbmi. My contributions included exploratory data analysis and a ",e.createElement("a",{href:"https://github.com/huangy6/beat_pd/tree/keller-mark/more-snakemake"},"snakemake pipeline")," for our team's feature extraction and training code.")},{title:"ExploSig",slug:"explosig",img:"explosig.png",description:e.createElement("p",null,"As an undergraduate research assistant in the ",e.createElement("a",{href:"https://lrgr.io"},"Leiserson Research Group")," at the University of Maryland, I led the development of ",e.createElement("a",{href:"https://github.com/lrgr/explosig"},"ExploSig"),", a web-based tool to enable exploration and interactive visualization of mutation signatures in human cancer and their etiology. ExploSig was presented as a late-breaking poster at ",e.createElement("a",{href:"https://recombccb.recomb2019.org/abstracts/"},"RECOMB 2019")," and won best poster at ",e.createElement("a",{href:"http://biovis.net/2019/index.html"},"BioVis@ISMB 2019"),".",e.createElement("br",null),e.createElement("br",null),"This work also included writing a pipeline for processing somatic mutation data and clinical data from TCGA, ICGC, PCAWG, and cBioPortal. This data processing is acknowledged in ",e.createElement("a",{href:"https://doi.org/10.1093/bioinformatics/btz340"},"Robinson et al. Bioinformatics 2019")," and ",e.createElement("a",{href:"http://doi.org/10.1007/978-3-030-17083-7_15"},"Sason et al. RECOMB 2019"),".")}];function S(){return e.createElement(u,{className:"main-grid"},e.createElement(g,null,e.createElement(E,{sm:4,md:1,className:"intro-img-col"},e.createElement("div",{className:"intro-img",style:{backgroundImage:"url(/assets/mark.08909fe4.png)"}})),e.createElement(E,{sm:4,md:7,className:"mk--pt-1"},e.createElement("p",null,"I am a third-year PhD student in the Bioinformatics and Integrative Genomics (",e.createElement("a",{href:"https://bigphd.hms.harvard.edu/",target:"_blank"},"BIG"),") program at Harvard Medical School, advised by Professor ",e.createElement("a",{href:"http://gehlenborglab.org/"},"Nils Gehlenborg"),". I graduated from the University of Maryland, College Park in 2019, where I studied Computer Science. My research interests include data visualization and computational biology."))),e.createElement(g,null,e.createElement(E,{sm:4,lg:6,className:"mk--pt-1 projects-col"},e.createElement("h3",null,"featured projects"),e.createElement(b,{className:"mk--pt-1"},M.map((t=>e.createElement(f,{key:t.slug,title:t.title},t.description))))),e.createElement(E,{sm:4,lg:6,className:"mk--pt-1 news-col"},e.createElement("h3",null,"news"),e.createElement("div",null,C.map((t=>e.createElement(g,{key:t.slug},e.createElement(E,{sm:1},t.date),e.createElement(E,{sm:3},t.description))))))))}function I(){return e.useLayoutEffect((()=>{if(CSS&&CSS.paintWorklet){const e=new Blob(["\n            registerPaint('smooth-corners', class {\n                static get inputProperties() {\n                    return [\n                        '--smooth-corners'\n                    ]\n                }\n                paint(ctx, geom, properties) {\n                    const c = properties.get('--smooth-corners').toString()\n            \n                    ctx.fillStyle = 'black'\n            \n                    const n = c\n                    let m = n\n                    if (n > 100) m = 100\n                    if (n < 0.00000000001) m = 0.00000000001\n                    const r = geom.width / 2\n                    const w = geom.width / 2\n                    const h = geom.height / 2\n            \n                    ctx.beginPath();\n            \n                    for (let i = 0; i < (2*r+1); i++) {\n                        const x = (i-r) + w\n                        const y = (Math.pow(Math.abs(Math.pow(r,m)-Math.pow(Math.abs(i-r),m)),1/m)) + h\n            \n                        if (i == 0)\n                            ctx.moveTo(x, y)\n                        else\n                            ctx.lineTo(x, y)\n                    }\n            \n                    for (let i = (2*r); i < (4*r+1); i++) {\n                        const x = (3*r-i) + w\n                        const y = (-Math.pow(Math.abs(Math.pow(r,m)-Math.pow(Math.abs(3*r-i),m)),1/m)) + h\n                        ctx.lineTo(x, y)\n                    }\n            \n                    ctx.closePath()\n                    ctx.fill()\n                }\n            })\n            "],{type:"text/javascript"});CSS.paintWorklet.addModule(URL.createObjectURL(e))}}),[]),e.createElement(w,null,e.createElement(x,null),e.createElement(y,null,e.createElement(k,{path:"/blog"},e.createElement("p",null,"Coming soon.")),e.createElement(k,{path:"/photos"},e.createElement("p",null,"Coming soon. For now some photos can be viewed on ",e.createElement("a",{href:"https://www.flickr.com/photos/83712651@N04/"},"Flickr"),".")),e.createElement(k,{path:"/"},e.createElement(S,null))))}v.render(e.createElement(e.StrictMode,null,e.createElement(I,null)),document.getElementById("root"));
